src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -40,6 +36,19 @@ public CommandResult execute() {
         return new CommandResult(getMessageForPersonListShownSummary(personsFound), personsFound);
     }
 
+    /**
+     *
+     * @param SensitiveKeywords for searching
+     * @return list of case-insensitive keywords
+     */
+    private static Set<String> convertToCaseInsensitive (Collection<String> SensitiveKeywords){
```

Comment:
According to the [Java Coding Standards](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#naming-conventions), variable names should be in camelCase.